docs/5_2_scheduled_execution_api.adoc

```diff
@@ -8,379 +8,125 @@ The simulation algorithm signals to the FMU that it supports and has recognized
+Special to Scheduled Execution, the following additional constrains apply to the state <<Initialized>>.
+The FMU enters state <<Terminated>> only after all other tasks related to the computation of model partitions of this FMU have ended.
+After <<fmi3Terminate>> has been called no new tasks can be started (e.g. related to <<outputClock>> ticks) and all other function calls for this FMU must return <<fmi3Error>> until the state <<Terminated>> is reached.
 
-===== State: Clock Activation Mode
+==== State: Clock Activation Mode [[ClockActivationMode,*Clock Activation Mode*]]
 
-The FMU enters this state when the simulation algorithm calls <<fmi3ExitInitializationMode>> in state *Initialization Mode* or <<fmi3ExitConfigurationMode>> in state *Reconfiguration Mode*.
+The FMU enters this state when the simulation algorithm calls <<fmi3ExitInitializationMode>> in state <<InitializationMode>> or <<fmi3ExitConfigurationMode>> in state <<ReconfigurationMode>>.
 
 In this state the simulation algorithm can create multiple concurrent tasks related to an FMU and in each task the simulation algorithm can activate one or multiple <<inputClock,`input clocks`>> of an FMU based on the defined <<clock>> properties via a <<fmi3ActivateModelPartition>> call for each <<clock>>.
 
-Allowed Function Calls::
-
-[[fmi3ActivateModelPartition, `fmi3ActivateModelPartition`]]
-<<fmi3ActivateModelPartition>>::
-
-Each <<fmi3ActivateModelPartition>> call is now associated to the computation of a (publicly disclosed, externally controlled) model partition of the model and therefore to a single defined <<inputClock>>.
-
-[source, C]
-----
-include::../headers/fmi3FunctionTypes.h[tag=ActivateModelPartition]
-----
-
-The <<fmi3ActivateModelPartition>> function has the following arguments:
+[cols="2,1",options="header",]
+|====
+|Equations and Actions
+|Functions Influencing Equations
 
-- `fmi3Instance instance`: same meaning as for other `fmi3*` functions
+|Set <<tunable>> <<parameter,`parameters`>> latexmath:[\mathbf{p}_{\mathit{variability == tunable}}] .
+|`fmi3Set{VariableType}`
 
-- `fmi3ValueReference` <<clockReference>>: <<valueReference>> of an <<inputClock>> variable defined in the <<modelDescription.xml>> which shall be activated
+|Set discrete-time <<input,`inputs`>> latexmath:[\mathbf{u}_{d}(\mathbf{t})].
+|`fmi3Set{VariableType}`
 
-- `size_t` <<clockElementIndex>>:: The element index of the <<inputClock>> variable which shall be activated.
-For a scalar <<clock>> variable this must be 0; for array <<clock>> variables, the element clock to activate is specified using the 1-based element index.
-Using the element index 0 means all elements of the <<clock>> variable.
-(Note: If an array has more than one dimension the indices are serialized in the same order as defined for values in <<serialization-of_variables>>).
+|Get values of variables latexmath:[\mathbf{v}(\mathbf{t})].
+|`fmi3Get{VariableType}`
```

Comment:
This order violates the calling sequence, should be moved after fmi3activateModelPartition!

Reply:
see above